pkg/proto/nbcontract/v1/imdsrestrictionconfig.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package nbcontract.v1;
+
+message IMDSRestrictionConfig {
```

Comment:
Technically everything is a `Config`. Do we want to use it for every struct?

Reply:
This was brought up earlier too.
One example to keep the `Config` is that for this kind of case.
If there is not `Config`, it will conflict.
```
message HTTPProxyConfig {
  string http_proxy = 1;
  string https_proxy = 2;
}
```
We can also change the http_proxy to http_proxy_setting or so.
Adding a `Config` is not the only solution, but more for consistency.

Happy to adopt another better way.